maapcopcguide (2).ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<ol><li>search_data -&gt; <code>search_data</code></li><li>The temporal argument defines the temporal range for... Is that an incomplete sentence?</li></ol><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>